modules/web/js/ballerina/utils/ace-worker.js

```diff
@@ -19,15 +19,20 @@
 import backends from 'bal_configs/backends';
 // This import defines ace/worker/mirror so we can ace.require ace/worker/mirror later
 import './ace-mirror-worker';
-ace.define('ace/worker/ballerina', ['require', 'exports', 'module'], (acequire, exports, module) => {
+
+const ace = global.ace;
+
+ace.define('ace/worker/ballerina', ['require', 'exports'], (acequire, exports) => {
```

Comment:
@roonyh , please check if this is wrong ? ace wasnt defined earlier. Got the value through global.